qiita_db/test/test_study.py

```diff
@@ -179,6 +179,54 @@ def _change_processed_data_status(self, new_status):
             "UPDATE qiita.processed_data SET processed_data_status_id = %s",
             (id_status,))
 
+    def test_get_info(self):
```

Comment:
You added a new test but it is still testing 1 single study, could you test multiple? `Study.get_info([1, 2])`